astro/src/content/blog/i-just-want-authentication-to-work.mdx

```diff
@@ -0,0 +1,77 @@
+---
+publish_date: 2023-11-17
+title: I Just Want Authentication To Work
+description: When I am writing an Application the last thing that I care about is setting up Authentication, I just want it to work!
+authors: Alex Patterson
+image: /img/blogs/i-just-want-authentication-to-work/i-just-want-auth-to-work.png
+categories: Company, News
+tags: award, security mag
+excerpt_separator: "{/* more */}"
+---
+
+Authentication is the process of verifying the identity of a user or entity. It is a fundamental security measure that is used in a wide variety of applications, including web applications, mobile apps, and APIs. The thing is as a developer with over 20 years of experience, I just don't care about figuring it out! I just want it to work!!
+
+{/* more */}
+
+There are many different ways to implement authentication, but the most common approach is to use a username and password. Yep, just like back in 1960 when [The Everly Brothers](https://youtu.be/KtY3L5tDN_E?si=BATB3BzJJFpWsHmB) had one of the top songs out there, this method is still most popular today. Just as the Everly Brothers' time at the top of the charts has passed, so has the time for passwords. I don't want to hear about them any more!
+
+A more modern approach is to use a single sign-on (SSO) solution. SSO solutions allow users to log in to multiple applications using a single set of credentials. This can be convenient for users, as they do not have to remember multiple usernames and passwords. This method came about around the same time Eminem hit the [billboard top 100](https://youtu.be/6mPCy9LMxTI?si=5A2I-N1neb_8rGOl) for the first time. SSO often has the problem of including the first login requirement of using a username and password still to this day. Even Eminem made some changes and remains relevant!
+
+My point is that authentication is an important security measure, but it can also be complex to implement. This is why we have been hanging on to these old methods so long! When you are creating a new application (unless it is one for authentication) the last thing you care about is trying to roll your own. There are many different authentication methods to choose from, and each method has its own advantages and disadvantages. Additionally, authentication needs to be implemented in a secure way to prevent attackers from gaining unauthorized access to applications. All I care about is that my data is secure when creating the next [Napster](https://youtu.be/CKrdsGdLVQ8?si=bTp4DHKS4tlpQzfW) so everyone can share their 1990's music with me again!
```

Comment:
```suggestion
My point is that authentication is an important security measure, but it can also be complex to implement. This is why we have been hanging on to these old methods so long! When you are creating a new application (unless it is an *authentication* application), the last thing you care about is user login. There are many different authentication methods to choose from, and each method has its own advantages and disadvantages. Additionally, authentication needs to be implemented in a secure way to prevent attackers from gaining unauthorized access to applications. 

But, again, all I care about is that my data is secure when creating the next [Napster](https://youtu.be/CKrdsGdLVQ8?si=bTp4DHKS4tlpQzfW) so everyone can share their 1990's music with me again!
```